jenkins-jobs/pipelines/connector_tests_pipeline.groovy

```diff
@@ -0,0 +1,151 @@
+                                booleanParam(name: 'PRODUCT_BUILD', value: params.PRODUCT_BUILD)
+                        ]
+                    }
+                }
+
+                stage('Invoke_postgresql') {
+                    when {
+                        expression { params.POSTGRESQL_TEST }
+                    }
+                    steps {
+                        build job: 'connector-debezium-postgresql-matrix-test', parameters: [
+                                string(name: 'REPOSITORY_CORE', value: params.REPOSITORY_CORE),
+                                string(name: 'BRANCH_CORE', value: params.BRANCH_CORE),
+                                string(name: 'SOURCE_URL', value: params.SOURCE_URL),
+                                booleanParam(name: 'PRODUCT_BUILD', value: params.PRODUCT_BUILD)
+                        ]
+                    }
+                }
+
+                stage('Invoke_sqlserver') {
+                    when {
+                        expression { params.SQLSERVER_TEST }
+                    }
+                    steps {
+                        build job: 'connector-debezium-sqlserver-matrix-test', parameters: [
+                                string(name: 'REPOSITORY_CORE', value: params.REPOSITORY_CORE),
+                                string(name: 'BRANCH_CORE', value: params.BRANCH_CORE),
+                                string(name: 'SOURCE_URL', value: params.SOURCE_URL),
+                                booleanParam(name: 'PRODUCT_BUILD', value: params.PRODUCT_BUILD)
+                        ]
+                    }
+                }
+            }
+        }
+
+        stage('Set_label') {
+            steps {
+                script {
+                    // check if all passed and store ran builds
+                    def child_builds = []
+                    def databases = ['db2', 'mongodb', 'mysql', 'oracle', 'postgresql', 'sqlserver']
+                    for (db in databases) {
+                        def run_test_arg = db.toUpperCase() + "_TEST"
+                        def job_name = "connector-debezium-" + db + "-matrix-test"
+                        if (!params[run_test_arg]) {
+                            continue
+                        }
+                        def build = jenkins.model.Jenkins.instance.getItem(job_name).lastBuild
+                        if(build.getResult() != hudson.model.Result.SUCCESS) {
+                            println "Not all ran tests passed, skipping label"
+                            return
+                        } else {
+                            child_builds << build
+                        }
+                    }
+                    if (!child_builds) {
+                        return
+                    }
+
+                    // determine labels for builds to set
+                    def label = params.LABEL
+                    def label_db2 = label
```

Comment:
this seems redundant. There is no difference in logic for naming DB2 build

Reply:
I left this, since DB2 can be on different repository and different branch. If this is to be deleted, so is at leasd db2 branch parameter I guess

Reply:
Technically yes it could,  for the branch it would make sense